src/ai/backend/web/proxy.py

```diff
@@ -164,7 +162,10 @@ async def web_handler(request: web.Request, *, is_anonymous=False) -> web.Stream
             raise RuntimeError("'pipeline' config must be set to handle pipeline requests.")
         endpoint = pipeline_config["endpoint"]
         log.info(f"WEB_HANDLER: {request.path} -> {endpoint}/{real_path}")
-        api_session = await asyncio.shield(get_api_session(request, endpoint))
+        if real_path.rstrip("/") == "login":
+            api_session = await asyncio.shield(get_api_session(request, endpoint))
+        else:
+            api_session = await asyncio.shield(get_anonymous_session(request, endpoint))
```

Comment:
Is there another way to identify it as a login path? Relying on the `real_path` could be fragile.